uxarray/plot/accessor.py

```diff
@@ -31,7 +31,16 @@ def __init__(self, uxgrid: Grid) -> None:
         self._uxgrid = uxgrid
 
     def __call__(self, **kwargs) -> Any:
-        return self.edges(**kwargs)
+        # Decide what to plot based on what's in the Grid
+        if self._uxgrid.n_face > 0:
+            # If we have faces, use the existing edges plotting
+            return self.edges(**kwargs)
+        elif self._uxgrid.n_edge > 0:
```

Comment:
While I get these checks here, is it practically possible to have grids where we wouldn't have faces but edges, or not even edges but nodes?

Reply:
Yes, sometimes there are point clouds or simply railway lines etc. that geopandas support reading. It is not common, but may be useful in some cases.